### PR TITLE
[zh-cn] sync kube-scheduler-config.v1.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/reference/config-api/kube-scheduler-config.v1.md

### CODE REVIEW

File: content/zh-cn/docs/reference/config-api/kube-scheduler-config.v1.md [hunk 1]
```diff
@@ -25,6 +25,256 @@ auto_generated: true
...
+**出现在：**
+
+- [KubeSchedulerConfiguration](#kubescheduler-config-k8s-io-v1beta3-KubeSchedulerConfiguration)
+
+- [KubeSchedulerConfiguration](#kubescheduler-config-k8s-io-v1-KubeSchedulerConfiguration)
+
+<!--
+DebuggingConfiguration holds configuration for Debugging related features.
+-->
+<p>DebuggingConfiguration 保存与调试功能相关的配置。</p>
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/kube-scheduler-config.v1.md:122]:
```suggestion
<p>DebuggingConfiguration 包含与调试功能相关的配置。</p>
```

File: content/zh-cn/docs/reference/config-api/kube-scheduler-config.v1.md [hunk 2]
```diff
@@ -262,6 +512,24 @@ with the extender. These extenders are shared by all scheduler profiles.
...
+</td>
+<td>
+   <p>
+   <!--
+   DelayCacheUntilActive specifies when to start caching. If this is true and leader election is enabled,
+   the scheduler will wait to fill informer caches until it is the leader. Doing so will have slower
+   failover with the benefit of lower memory overhead while waiting to become leader.
+   Defaults to false.
+   -->
+   DelayCacheUntilActive 指定何时开始缓存。如果这是真的并且启用了 leader 选举，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/kube-scheduler-config.v1.md:526]:
```suggestion
   DelayCacheUntilActive 指定何时开始缓存。如果字段设置为 true 并且启用了领导者选举，
```

File: content/zh-cn/docs/reference/config-api/kube-scheduler-config.v1.md [hunk 3]
```diff
@@ -262,6 +512,24 @@ with the extender. These extenders are shared by all scheduler profiles.
...
+   则调度程序将等待填充通知者缓存，直到它成为 leader，这样做会减慢故障转移速度，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/kube-scheduler-config.v1.md:527]:
```suggestion
   则调度程序将等待填充通知者缓存，直到它成为领导者，这样做会减慢故障转移速度，
```

**Review by ydFu — APPROVED** [review]
/lgtm

**Pull request merged**